src/app/containers/App/App.jsx

```diff
@@ -66,7 +66,17 @@ export const App = ({ routes, location, initialData, bbcOrigin }) => {
     }
   }, [routes, location.pathname]);
 
-  return renderRoutes(routes, { ...state, bbcOrigin });
+  const pathRef = useRef(null);
```

Comment:
I could be wrong but does this need to be a ref? Do we even need the const `pathRef`? 

I just tried this locally and I think we can remove this line, line 73 and then on line 75 we can have `setPath(pathRef.current);`

Meaning this could be simplified to:
```
  const [previousPath, setPath] = useState(null);

  useEffect(() => {
    return () => {
      setPath(location.pathname);
    };
  }, [location.pathname]);
```